### DISCUSSION TITLE
API Resep Makanan

### DISCUSSION THREAD

**Discussion by andriyfm** [farizdotid/DAFTAR-API-LOKAL-INDONESIA#323] [General]
Adakah temen-temen disini yang punya rest api tentang resep masakan yang lengkap?

**Comment by putra4648** [comment]
Ada sih, yang saya tahu Zomato, Spooncular, TheMealDB dan Edamam

**Comment by farizdotid** [comment]
resep khusus makanan Indonesia mas @andriyfm ?

**Comment by andriyfm** [comment]
> resep khusus makanan Indonesia mas @andriyfm ?
> 

Iyah mas khusus indonesia. Buat kbutuhan app umkm

**Comment by farizdotid** [comment]
> Iyah mas khusus indonesia. Buat kbutuhan app umkm

Barusan nyobain baca dokumentasi dari API berikut Zomato, Spooncular, TheMealDB dan Edamam ga ada filter by country gitu euy.

**Comment by putra4648** [comment]
Iya gk ada filter country masalahnya itu

**Comment by andriyfm** [comment]
Btw makasih. Ntr sya cobain dlu deh

**Comment by farizdotid** [comment]
Atau bikin sendiri mas dengan case hanya resep makanan Indonesia 😃

**Comment by andriyfm** [comment]
> Atau bikin sendiri mas dengan case hanya resep makanan Indonesia 😃 

Duh gak ada waktunya wuy, kpan2 lah nyobain bikin 😁

**Comment by farizdotid** [comment]
udah ada di nih API untuk masakan Indonesia mas @andriyfm . Boleh dicek disini ya : https://github.com/farizdotid/DAFTAR-API-LOKAL-INDONESIA#buku

**Comment by andriyfm** [comment]
> udah ada di nih API untuk masakan Indonesia mas @andriyfm . Boleh dicek disini ya : https://github.com/farizdotid/DAFTAR-API-LOKAL-INDONESIA#buku

Wih mantap, makasih banget @farizdotid langusung implemantasiin ke aplikasi nih. Makasih juga yang udah contribute @tomorisakura

**Comment by tomorisakura** [comment]
> > udah ada di nih API untuk masakan Indonesia mas @andriyfm . Boleh dicek disini ya : https://github.com/farizdotid/DAFTAR-API-LOKAL-INDONESIA#buku
> 
> Wih mantap, makasih banget @farizdotid langusung implemantasiin ke aplikasi nih. Makasih juga yang udah contribute @tomorisakura 

sama sama mas , semoga membantu ✌

**Comment by justirva09** [comment]
> udah ada di nih API untuk masakan Indonesia mas @andriyfm . Boleh dicek disini ya : https://github.com/farizdotid/DAFTAR-API-LOKAL-INDONESIA#buku

ini masih belum ada detail recipe nya kah? atau cuma nampilin preview2 nya aja ya?

**Comment by tomorisakura** [comment]
> > udah ada di nih API untuk masakan Indonesia mas @andriyfm . Boleh dicek disini ya : https://github.com/farizdotid/DAFTAR-API-LOKAL-INDONESIA#buku
> 
> ini masih belum ada detail recipe nya kah? atau cuma nampilin preview2 nya aja ya?

ada mas , coba baca di bagian endpoint usage, disitu ada recipe-detail

**Comment by justirva09** [comment]
ginikah resultnya? atau saya salah? @tomorisakura bisa bantu kasih example endpoint untuk dapetin detail resep ?
![image](https://user-images.githubusercontent.com/48151470/99662745-40790280-2a98-11eb-9545-5ce2611824dc.png)

**Comment by tomorisakura** [comment]
> ginikah resultnya? atau saya salah? @tomorisakura bisa bantu kasih example endpoint untuk dapetin detail resep ?
> ![image](https://user-images.githubusercontent.com/48151470/99662745-40790280-2a98-11eb-9545-5ce2611824dc.png)

ketika mas mau ngehit detail resep pastikan udah dapat **key** dari resepnya dan untuk penggunaan detail resep bisa hit `baseurl/api/recipe/:key` contohnya https://masak-apa.tomorisakura.vercel.app/api/recipe/resep-dimsum-ayam-seenak-restoran

**Comment by syahid-ann** [comment]
halo mas tomorisakura.. mas, aku kan  nyoba api yang dibuat sama mas nih, nah pas mau cari resep nya itu ada error seperti ini "Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource..." itu kenapa ya mas? terima kasih ..

**Comment by tomorisakura** [comment]
> halo mas tomorisakura.. mas, aku kan nyoba api yang dibuat sama mas nih, nah pas mau cari resep nya itu ada error seperti ini "Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource..." itu kenapa ya mas? terima kasih ..

Sepertinya terjadi kesalahan pada web induknya mas, saya juga nyobain nyari resep gabisa dan saya cek di web induknya terjadi trouble pas ngesearch resep

**Comment by fckveza** [comment]
api.vhtear.com

**Comment by FthRobby** [comment]
mas @tomorisakura ijin pake yo mas, buat tugas kuliah

**Comment by aldi-milano** [comment]
@tomorisakura kena CORS policy mas, kenapa ya?

**Comment by kisnak21** [comment]
@tomorisakura masih kena CORS policy mas, barusan nyoba. Smoga bisa fix mas

**Comment by aliefrahman** [comment]
coba ini -> https://www.edamam.com/

**Comment by aryomuzakki** [comment]
dari @tomorisakura barusan nyoba gabisa, sementara nemu yg bisa, ada di : https://github.com/Rauliqbal/masakapahariini-api
